### ISSUE TITLE
TypeError: Cannot read property 'confirmed' of undefined

### ISSUE THREAD

**Issue by ltfschoen** [paritytech/fether#379]
After clicking "Send" on the transaction page, I've started encountering this error whilst waiting for the transaction to process. I was able to replicate it by reloading the window whilst it was sending the transaction (i.e. View > Reload)

<img width="1118" alt="screen shot 2019-01-22 at 7 25 18 am" src="https://user-images.githubusercontent.com/6226175/51498610-401b6c00-1e1b-11e9-8008-af5d04a1f34c.png">

**Comment by ltfschoen** [comment]
Yes, I was only sending from Parity Signer accounts this morning.
I don't recall it happening straight away. I think the first time it happened I'd sent the transaction, and then did some other things, and saw the error when I cam back.
But another time I reloaded the page after sending the transaction and that triggered the error.
I'll have to check again.

**Comment by ltfschoen** [comment]
I just tried to replicate it again. Each time I try and send a transaction from a Parity Signer account, after scanning all the QR codes and clicking "Send", if I go to Electron menu "View > Reload" it displays the error.

I'm not sure if it's a related issue or not, but I tried sending the same "amount" with same "from" and "to" accounts a second time and it displayed the following error when I tried to click "Send". I think this is a concern because it didn't show the error in the UI, I just couldn't do anything other than to click the arrow icon:

<img width="1119" alt="screen shot 2019-01-23 at 7 06 58 am" src="https://user-images.githubusercontent.com/6226175/51562505-124b2b80-1ede-11e9-83d6-d79109a3f836.png">
